includes/class-wc-payments-platform-checkout-button-handler.php

```diff
@@ -94,14 +94,30 @@ public function scripts() {
 			return;
 		}
 
+		$platform_express_button_params = [
+			'button' => $this->get_button_settings(),
+		];
+
+		$script_src_url    = plugins_url( 'dist/platform-checkout-express-button.js', WCPAY_PLUGIN_FILE );
+		$script_asset_path = WCPAY_ABSPATH . 'dist/platform-checkout-express-button.asset.php';
+		$script_asset      = file_exists( $script_asset_path ) ? require_once $script_asset_path : [ 'dependencies' => [] ];
+
+		wp_register_script( 'WCPAY_PLATFORM_CHECKOUT_EXPRESS_BUTTON', $script_src_url, $script_asset['dependencies'], WC_Payments::get_file_version( 'dist/platform-checkout-express-button.js' ), true );
+
+		wp_localize_script( 'WCPAY_PLATFORM_CHECKOUT_EXPRESS_BUTTON', 'wcpayWooPayExpressParams', $platform_express_button_params );
+
+		wp_set_script_translations( 'WCPAY_PLATFORM_CHECKOUT_EXPRESS_BUTTON', 'woocommerce-payments' );
+
+		wp_enqueue_script( 'WCPAY_PLATFORM_CHECKOUT_EXPRESS_BUTTON' );
+
 		wp_register_style(
-			'wcpay-platform-checkout-express-button-styles',
-			plugins_url( 'includes/platform-checkout/assets/css/platform-checkout-express-button.css', WCPAY_PLUGIN_FILE ),
+			'WCPAY_PLATFORM_CHECKOUT',
```

Comment:
Is it required to register the styles as this CSS file is already registered [here](https://github.com/Automattic/woocommerce-payments/blob/add/1427-style-woopay-express-checkout-button/includes/platform-checkout-user/class-platform-checkout-save-user.php#L44-L49)? :thinking:

Reply:
I understand why you're putting this in in anticipation of the stylesheet not always being present. Do we have an issue to look in to that?

Reply:
> Do we have an issue to look in to that?

Not yet. I can make one, though.

Reply:
Actually, it looks like this could be fixed along with #5118. I'll leave a comment there noting that the script/style files are being included on all pages.